Ports/mrsh/patches/ReadMe.md

```diff
@@ -0,0 +1,22 @@
+# Patches for mrsh on SerenityOS
+
+## `0001-Fix-version-script.patch`
+
+Fix version script
+
+
+## `0002-Hardcode-default-path-because-confstr-is-missing.patch`
+
+Hardcode default path because `confstr` is missing
+
+
+## `0003-Workaround-warnings.patch`
+
+Workaround warnings
+
+
+## `0004-Disable-glob.patch`
+
+Disable glob
+
+
```

Comment:
The patches could probably use a bit more of a description. Why are we disabling glob? What warnings do we workaround? What's wrong with the default version script?

Reply:
I tried to be a bit more descriptive about the patches, but as I picked this port up from
someone else, I'm not 100% sure how to best explain the patches.